ci/scripts/utils/launch_java_agent.sh

```diff
@@ -74,7 +74,7 @@ host=$(hostname)
 
 source "${HOMEGFS_}/ush/detect_machine.sh"
 case ${MACHINE_ID} in
-  hera | orion | hercules | wcoss2 | gaea)
+  hera | orion | hercules | wcoss2 | gaea | gaeac6 )
```

Comment:
```suggestion
  hera | orion | hercules | wcoss2 | gaeac5 | gaeac6 )
```

Reply:
Ok, even though we are not using gaeac5 lets put this in anyway.